src/main/java/dev/jbang/catalog/Alias.java

```diff
@@ -44,6 +44,8 @@ public class Alias extends CatalogItem {
 	public final Boolean nativeImage;
 	@SerializedName(value = "native-options")
 	public final List<String> nativeOptions;
+	@SerializedName(value = "source-type")
```

Comment:
this is new feature,right? being able to set source type on aliases?

Reply:
Yes, although I would say it was more a _missing_ feature, because in general all command line options should be made available in aliases as much as possible. I guess this one just slipped through the cracks.